src/Waargonaut/Types/JString.hs

```diff
@@ -102,18 +100,16 @@ instance AsJString [JChar HeXDigit] where
 
 instance AsJString String where
   _JString = prism
-    (\(JString' cx) -> V.toList $ (_JChar #) <$> cx)
-    (\x -> JString' . V.fromList <$> traverse (note x . (^? _JChar)) x)
+    (\(JString' cx) -> V.toList $ jCharToChar <$> cx)
+    (\x -> JString' . V.fromList <$> traverse (note x . charToJChar) x)
 
 -- | Prism between a 'JString' and 'Text'.
 --
 -- JSON strings a wider range of encodings than 'Text' and to be consistent with
 -- the 'Text' type, these invalid types are replaced with a placeholder value.
 --
-_JStringText :: Prism' JString Text
-_JStringText = prism
-  (JString' . V.fromList . fmap utf8CharToJChar . Text.unpack)
-  (\x -> first (const x) . Text.decodeUtf8' . BS8.toStrict . BS8.pack . review _JString $ x)
```

Comment:
Upon reflection, this is the software development equivalent of: "I'll walk it through Mayan first"